README.md

```diff
@@ -333,6 +336,32 @@ web browser.
 
 ![turing ui list router](./docs/assets/turing_ui_router_list.png)
 
+## Installing prerequisites on Kubernetes using the init chart
+
+This is the recommended way to install the prerequisites on a production Kubernetes cluster. The `turing-init` Helm chart will install the required software needed to run Turing, namely:
+
+1. [Knative Serving](https://knative.dev/docs/serving/)
+2. [Istio](https://istio.io/)
+3. [Spark on K8s Operator](https://github.com/GoogleCloudPlatform/spark-on-k8s-operator)
+
+To install the required components on your Kubernetes cluster, issue the following command:
+
+```bash
+helm upgrade -n infrastructure turing-init infra/charts/turing-init \
+    --set image.registry="<image registry>/" \
+    --set image.repository="<repository>" \
+    --set image.tag="<tag version>" \
+    --install \
+    --wait
+```
```

Comment:
Can we expand this section a bit and include the success criteria. I.e. what resources users expect to see in the cluster after provisioning is successful.

Reply:
Ya ok